### PR TITLE
Added a repeatable background to create a cool pattern

### PR DESCRIPTION
**Pull request by ThunderDev1**
What do you think ?

### CODE REVIEW

File: client/src/layouts/BaseLayout.scss [hunk 1]
```diff
@@ -2,7 +2,9 @@
 @import 'styles/helpers';
 
 body {
-  background-color: $color-primary;
+  background-image: url('../images/backgound-light-grey.png');
```
Comment (bellangerq) [inline client/src/layouts/BaseLayout.scss]: Remove all unused images and rename the remaining one `background.png` instead of `backgound-light-grey.png`.

File: client/src/layouts/BaseLayout.scss [hunk 2]
```diff
@@ -2,7 +2,9 @@
...
+  background-repeat: repeat-x repeat-y;
```
Comment (bellangerq) [inline client/src/layouts/BaseLayout.scss]: `repeat-x repeat-y` is a wrong property value. The correct one is: `background-repeat: repeat.` And you can even combine it with other `background-*` properties:

```css
// This...
background-image: url(...);
background-repeat: repeat;

// ...is the same as
background: url(...) repeat;
```

But by the way, `repeat` is the default so you can just remove this line.

File: client/src/layouts/BaseLayout.scss [hunk 3]
```diff
@@ -2,7 +2,9 @@
...
+  background-size: 90px;
```
Comment (bellangerq) [inline client/src/layouts/BaseLayout.scss]: Never use `px`!

File: client/src/layouts/BaseLayout.scss [hunk 4]
```diff
@@ -15,8 +17,9 @@ body {
   flex-direction: column;
   max-width: $breakpoint-desktop;
   margin: 0 auto;
-  min-height: 100vh;
+  min-height: 95vh;
```
Comment (bellangerq) [inline client/src/layouts/BaseLayout.scss]: Why did you change this? 🤔

File: client/src/layouts/BaseLayout.scss [hunk 5]
```diff
@@ -15,8 +17,9 @@ body {
...
   padding: $spacing-large;
+  border: 2px solid lightgrey;
```
Comment (bellangerq) [inline client/src/layouts/BaseLayout.scss]: Maybe add a `$border-large` variable?

File: client/src/layouts/BaseLayout.scss (hunk 1 above)
Comment (ThunderDev1) [reply client/src/layouts/BaseLayout.scss]: I left them so we could try different layouts, but if you're ok with this version it's fine by me.

File: client/src/layouts/BaseLayout.scss (hunk 4 above)
Comment (ThunderDev1) [reply client/src/layouts/BaseLayout.scss]: At 100vh the page is too high, and adds a sroll bar

Comment (bellangerq) [reply client/src/layouts/BaseLayout.scss]: Fixed in #106.

File: client/src/layouts/BaseLayout.scss [hunk 6]
```diff
@@ -2,7 +2,8 @@
 @import 'styles/helpers';
 
 body {
-  background-color: $color-primary;
+  background: url('../images/backgound.png');
```
Comment (bellangerq) [inline client/src/layouts/BaseLayout.scss]: It's missing an `r`!

Comment (ThunderDev1) [reply client/src/layouts/BaseLayout.scss]: Aha nice catch
